install/updates/20-aci.update

```diff
@@ -25,6 +25,10 @@ add:aci:(targetfilter="(objectclass=domain)")(targetattr="objectclass || dc || i
 dn: $SUFFIX
 add:aci:(targetattr="parentid")(version 3.0; acl "Anonymous read access to parentID information"; allow(read, search, compare) userdn = "ldap:///anyone";)
 
+# Read access to altSecurityIdentities to allow filter optimizations in 389-ds
+dn: $SUFFIX
```

Comment:
Does certmap only apply to users? If so it would be better to put this into the user container otherwise it would get evaluated with every request.

Reply:
Certmap rules can be associated with ID Overrides too, I think. For altSecurityIdentities, these, technically, can be present in any bindable object.